occurrences/schema.py

```diff
@@ -432,6 +435,25 @@ class StudyGroupInput(InputObjectType):
     study_level = StudyLevelEnum()
 
 
+def verify_captcha(key):
+    if not key:
+        raise CaptchaValidationFailedError("Missing captcha verification data")
+    secret_key = settings.RECAPTCHA_SECRET_KEY
+    verify_url = settings.RECAPTCHA_VALIDATION_URL
+
+    # captcha verification
+    data = {"response": key, "secret": secret_key}
+    resp = requests.post(verify_url, data=data)
```

Comment:
Requests' requests (hehe) don't have a timeout at all by default, so it is [recommended](https://requests.readthedocs.io/en/master/user/quickstart/#timeouts) to always set one.

Reply:
Fixed, we have other places that also use `requests` but I'll fix them later in a separate commit